From 887e67192662fb0887eb9cb402027d5eb7457b49 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Wed, 26 Oct 2011 20:46:12 +0800
Subject: [PATCH 0496/2508] ENGR00160802-1 arch/arm: Add BIT_INT_REASON reg
 definition in mxc_vpu.h

The BIT_INT_REASON register is needed in vpu driver code, so add it here.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mxc_vpu.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/mxc_vpu.h b/arch/arm/plat-mxc/include/mach/mxc_vpu.h
index 0f81fb9..06fc69a 100755
--- a/arch/arm/plat-mxc/include/mach/mxc_vpu.h
+++ b/arch/arm/plat-mxc/include/mach/mxc_vpu.h
@@ -60,6 +60,7 @@ struct vpu_mem_desc {
 #define BIT_CODE_DOWN			0x004
 #define	BIT_INT_CLEAR			0x00C
 #define	BIT_INT_STATUS			0x010
+#define BIT_INT_REASON			0x174
 
 #define MJPEG_PIC_STATUS_REG		0x3004
 
-- 
1.7.7.4

